doc/RecordingSearch.xml

```diff
@@ -1389,6 +1389,67 @@ http://www.onvif.org/ver10/tptz/ZoomSpaces/PositionGenericSpace
         </varlistentry>
       </variablelist>
     </section>
+    <section>
+      <title>FindRecordedSegments</title>
```

Comment:
Is this method needed for export functionality?

Reply:
I generally comes first, when the clients want to know what is available nad in what sections, that it can then export more intelligently based on what it already has, and what users need
For our precise usecase, it also permits creation of an HLS playlist from those segments. I believe it also enables creating DASH manifests or other similar techs